### PR TITLE
Add Korean docs link to landing page

### PR DESCRIPTION
**Pull request by wgordon17**

### CODE REVIEW

File: index-commercial.html [hunk 1]
```diff
@@ -323,8 +323,7 @@ <h2>OpenShift Container Platform <font size="-1">(日本語翻訳)</font></h2>
         <div class="col-sm-3 text-center">
           <h2>OpenShift Container Platform <font size="-1">(中文文档)</font></h2>
           <div class="list-group">
-            <p class="list-group-item-text">红帽的私有内部云应用程序部署和托管平台。
-</p>
+            <p class="list-group-item-text">红帽的私有内部云应用程序部署和托管平台。</p>
```
Comment (wgordon17) [inline index-commercial.html:326]: No content change, just moving the closing paragraph tag up

**Review by wgordon17 — COMMENTED** [review]
I suggest that we have a native Korean speaker validate the content provided by google translate

File: index-commercial.html [hunk 2]
```diff
@@ -338,6 +337,23 @@ <h2>OpenShift Container Platform <font size="-1">(中文文档)</font></h2>
           </div>
         </div>
 
+        <div class="col-sm-3 text-center">
+          <h2>OpenShift Container Platform <font size="-1">(한국어 번역)</font></h2>
```
[the remark this answers is not in the document]
Comment (ctomasko) [reply index-commercial.html:341]: Here's the original translations of the required fields:

"OpenShift Container Platform (Korean Documentation)"
"OpenShift Container Platform (한국어 문서)"
"Red Hat's private, on-premise cloud application deployment and hosting platform."
"Red Hat의 프라이빗 온프레미스 클라우드 애플리케이션 배포 및 호스팅 플랫폼입니다."

"Select Version"
"버전 선택"

[the remark this answers is not in the document]
Comment (wgordon17) [reply index-commercial.html]: Thank you @ctomasko, I've updated the content accordingly

**Comment by mbearer** [comment]
this lgtm should be ready to add for 4.10

**Comment by bobfuru** [comment]
@adellape PTAL?

**Comment by ctomasko** [comment]
Confirmed with Terry Chuang that the Korean linguist reviewed and approved the translations. LGTM. Please merge.

**Review by adellape — APPROVED** [review]

**Pull request merged**